pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -402,9 +423,23 @@ func NewLegacyRegistry() *LegacyRegistry {
 			}
 		})
 	registry.registerPriorityConfigProducer(EvenPodsSpreadPriority,
-		func(args ConfigProducerArgs, plugins *config.Plugins, _ *[]config.PluginConfig) {
+		func(args ConfigProducerArgs, plugins *config.Plugins, pluginConfig *[]config.PluginConfig) {
 			plugins.PreScore = appendToPluginSet(plugins.PreScore, podtopologyspread.Name, nil)
 			plugins.Score = appendToPluginSet(plugins.Score, podtopologyspread.Name, &args.Weight)
+			if feature.DefaultFeatureGate.Enabled(features.DefaultPodTopologySpread) {
+				// Configure plugin to use List default spreading, unless SelectorSpreadPriority was added.
+				for _, e := range *pluginConfig {
+					if e.Name == podtopologyspread.Name {
+						return
```

Comment:
trying to refresh my memory, this will not be populated via CC because we don't allow both policy and CC to be set at the same time (or do we? :)), and so this is only possible if the registry function for `SelectorSpreadPriority` was invoked first, correct?

Reply:
Correct, we only allow Policy or Plugins/PluginConfig, but not both.

The couple of producer functions are assuming that each other could run first.

Reply:
I would add comments clarifying that.